gcc/rust/lex/rust-lex.h

```diff
@@ -393,7 +389,6 @@ class Lexer
   int current_column;
   // Current character.
   Codepoint current_char;
-  Codepoint current_char32;
```

Comment:
This field was added to parse Unicode identifiers via https://github.com/Rust-GCC/gccrs/pull/2284 but is unnecessary now.